src/utils.ts

```
@@ -222,4 +242,4 @@ export function convertToTelegramButtons(buttons?: Button[] | null): InlineKeybo
 export function cleanText(text: string | undefined | null): string {
   if (!text) return '';
   return text.replace(/\u0000/g, '');
-}
\ No newline at end of file
+}
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)